cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -40,11 +40,12 @@ type MasterConfiguration struct {
 	KubernetesVersion string
 	CloudProvider     string
 	AuthorizationMode string
+	CertAltNames      []string
```

Comment:
We should start documenting these in this file like other APIs.

We can also explicitly list here which phases this input is used for.  May be easier to grok vs. looking at the doc.go files in each package.

CertAltNames: "A list of extra names to include in the Subject Alternate Name section of the API server certificate.  Use this if you will be accessing the server using some DNS name that isn't the host name of the node."